launch/cylinder_spawner.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
```

Comment:
Is this file necessary for the FT sensor, or is it just for testing the grasp? If it is not used in the FT sensor code, could you take it out of the PR?